index.js

```diff
@@ -1,24 +1,27 @@
-import { NativeModules } from 'react-native'
+import { NativeModules } from 'react-native';
 
-const { RNResponsysBridge } = NativeModules
+const { RNResponsysBridge } = NativeModules;
 
 const ResponsysBridge = {
   registerUserId: function (userId) {
     // Convert to string to prevent iOS conversion error
-    RNResponsysBridge.registerUserId('' + userId)
+    RNResponsysBridge.registerUserId('' + userId);
```

Comment:
This project is using [standard](https://standardjs.com/), that's why semicolons were missing.

Reply:
maybe, @diegotsi can add prettier config used in apps here too